pkgs/development/tools/gotestsum/default.nix

```
@@ -2,37 +2,39 @@
   lib,
   fetchFromGitHub,
   buildGoModule,
-}:
-buildGoModule rec {
-  pname = "gotestsum";
-  version = "1.10.1";
+}: let
```

is the file-wide reformat from the rfc formatter?
If we do this, will we get rid of the file-wide indent?
```suggestion
}: 
let
```

---

yeah this was a result of the new rfc formatter

---

but does adding the suggested line break avoid indenting the file?